src/clr-angular/popover/dropdown/dropdown-item.ts

```diff
@@ -27,7 +29,8 @@ export class ClrDropdownItem implements AfterViewInit {
     private el: ElementRef<HTMLElement>,
     private _dropdownService: RootDropdownService,
     private renderer: Renderer2,
-    private focusableItem: FocusableItem
+    private focusableItem: FocusableItem,
+    @Inject(UNIQUE_ID) public id: string
```

Comment:
The dropdown focus handler service already has a unique_id defined there, do we need it here again? Can't we just lean on that instead of having two unique ids?

Reply:
You are right. I just didn't want to go to this rabbit hole of refactoring. 

I think that the`FocusableItem` doesn't need to have the `id` it's never used anywhere. And it's hiding it deep inside the codebase. It's a component responsibility to handle the HTML id. 

But because I don't want to make it a braking change now. Can we mark `FocusableItem.id` for depreciation in the next version? 

```ts
export abstract class FocusableItem {
  /** @deprecated since 3.0, remove in 4.0 */
  id: string;
  // ....
```

Right now the code below is there just to handle this unused anywhere else field. 
```ts
  @Input('id')
  set dropdownItemId(value: string) {
    this.focusableItem.id = value;
  }
  get dropdownItemId() {
    return this.focusableItem.id;
  }
```